examples/src/main/java/com/jagrosh/jdautilities/examples/command/ServerinfoCommand.java

```diff
@@ -71,7 +71,7 @@ protected void execute(CommandEvent event)
         }
         String str = LINESTART + "ID: **" + guild.getId() + "**\n"
                 + LINESTART + "Owner: " + (owner == null ? "Unknown" : "**" + owner.getUser().getName() + "**#" + owner.getUser().getDiscriminator()) + "\n"
-                + LINESTART + "Location: " + (guild.getRegion().getEmoji().isEmpty() ? NO_REGION : guild.getRegion().getEmoji()) + " **" + guild.getRegion().getName() + "**\n"
+                + LINESTART + "Location: " + (guild.getVoiceChannels().get(0).getRegion().getEmoji().isEmpty() ? NO_REGION : guild.getVoiceChannels().get(0).getRegion().getEmoji()) + " **" + guild.getVoiceChannels().get(0).getRegion().getName() + "**\n"
```

Comment:
> Guilds no longer have the Region option. Use VoiceChannel.getRegion() instead.
Because of this change we have to have this work around, it will throw an error if there is no voice channels.
Maybe worth just removing the line entirely?

Reply:
Yeah since this is _server info_ not really needed plus this would almost always return automatic which would not have an emoji.